libs/ui-watt/src/lib/components/field/watt-field.component.scss

```diff
@@ -0,0 +1,69 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+@use "@energinet-datahub/watt/utils" as watt;
+
+watt-field {
+  --watt-text-color: var(--watt-on-light-high-emphasis);
+  --watt-disabled-text-label-color: var(--watt-on-light-low-emphasis);
+  --watt-field-disabled-background-color: var(--watt-color-neutral-grey-300s);
+  --watt-input-border-radius: 4px;
+  --watt-border-color: var(--watt-on-light-low-emphasis);
+  --watt-label-text-color: var(--watt-text-color);
+
+  label {
+    color: var(--watt-label-text-color);
+    // :has should be implmeneted in firefox before we go live
+    // https://connect.mozilla.org/t5/ideas/when-is-has-css-selector-going-to-be-fully-implemented-in/idi-p/23794/page/3#comments
+    &:has(.disabled) .label {
+      color: var(--watt-disabled-text-label-color);
+    }
+
+    &:has(.ng-dirty.ng-invalid) .label {
+      color: var(--watt-color-state-danger);
+    }
+
+    &:has(.ng-dirty.ng-invalid, [required]) .label::after {
+      content: " *";
+    }
+
+    .watt-field {
```

Comment:
nitpick: is `watt-field-input` a better name? i got confused thinking this was the component selector.

Reply:
But it would also contain a select at somepoint?

Reply:
Renamed it to .watt-field-wrapper